docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -97,6 +100,40 @@ http_codec_options
   <config_cluster_manager_cluster_http_codec_options>` option. See the comment there about
   disabling HTTP/2 header compression.
 
+.. _config_http_conn_man_http2_settings:
+
+http2_settings
+  *(optional, object)* Additional HTTP/2 codec options that are passed directly to the HTTP/2 codec. 
+  Currently supported settings are:
+
+  hpack_table_size
+    *(optional, integer)* `Maximum Table Size`_ (in octets) that the encoder is permitted to use for
+    the dynamic HPACK table. Valid values range from 0 to 16777216 (2^24) and defaults to 4096,
+    with 0 effectively disables header compression.
+
+    NOTE: A 16MiB (2^24) maximum table size seems to be more than enough.
+
+  max_concurrent_streams
+    *(optional, integer)* `Maximum concurrent streams`_ allowed on one HTTP/2 connection. Valid values
+    range from 1 to 536870912 (2^29) and defaults to 1024.
+
+    NOTE: Total HTTP/2 streams is 2^31, one side (client/server) is 2^30. To be safe, we use 2^29.
+
+  initial_window_size
+    *(optional, integer)* `Initial flow-control window`_ size. Valid values range from 65535
+    (HTTP/2 default window size, also minimum) to 2147483647 (2^31 - 1, also HTTP/2 maximum)
+    and defaults to 268435456 (256 * 1024 * 1024).
+
+    NOTE: 65535 is the initial window size from HTTP/2 spec. We only support increase the default window
+    size now, so it's also the minimum.
+
+  These are the same options available in the upstream cluster :ref:`http2_settings
+  <config_cluster_manager_cluster_http2_settings>` option.
+
+  .. _Maximum Table Size: http://httpwg.org/specs/rfc7541.html#maximum.table.size
+  .. _Maximum concurrent streams: http://httpwg.org/specs/rfc7540.html#rfc.section.5.1.2
+  .. _Initial flow-control window: http://httpwg.org/specs/rfc7540.html#InitialWindowSize
```

Comment:
Nit: could you change link anchors to `#rfc.section.X.X.X` to make things consistent?